dags/lib/doc/qa.py

```diff
@@ -19,7 +19,8 @@
 - Vérifier qu'aucune entité n'est dupliquée
 
 ### Différents tests
-- Table gnomad_genomes_v3
+- Table gnomad_genomes_v4
+- Table gnomad_joint_v4
```

Comment:
On ne veut pas tester gnomad_exomes_v4 aussi?

Reply:
@meek0 Corrige-moi mais je vois bel et bien gnomad_exomes_v4 dans le code.

Reply:
Je me suis fié à ce que je vois ici dans Zepplin
![image](https://github.com/user-attachments/assets/e763494c-ec95-4aae-867f-5d5d2aae6e4a)

Reply:
Ah! Je vois. Donc en fait on utilise juste la table publique `gnomad_joint_v4`, qui elle contient `genomes`, `exomes` et `joint`. Cédric avait chargé `gnomad_genomes_v4` mais finalement il a dû changer pour `gnomad_joint_v4`. Donc techniquement `gnomad_genomes_v4` pourrait être supprimée. @meek0 corrige-moi si je me trompe.

Reply:
oui, il y a exomes, genomes et joint

Reply:
Normal que la table gnomad_exomes_v4 ne soit pas disponible dans Zeppelin? Je créé un ticket?

Reply:
oui, tout devrait etre sous la table de joint as in `gnomad_joint_v4.ac_exomes`, `gnomad_joint_v4.ac_genomes`, `gnomad_joint_v4.ac_joint`, etc.

je pense que comme Laura a dit c'est ok de seulement check pour gnomad_joint_v4